app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for man_or_woman in people:
+        person_list.append(Person(name=man_or_woman["name"],
+                                  age=man_or_woman["age"]))
+    for man_or_woman in people:
+        if "wife" in man_or_woman and man_or_woman["wife"] is not None:
```

Comment:
You can try to make it shorter with `dict.get()` the same with husband

Reply:
How ?